classes/Language.php

```diff
@@ -1254,6 +1267,9 @@ public static function downloadXLFLanguagePack($locale, &$errors = [], $type = s
             return false;
         }
 
+        $language_pack_loader = new RemoteLanguagePackLoader(Version::buildFromString(_PS_VERSION_));
+        $url = $language_pack_loader->getLanguagePackUrl($locale);
```

Comment:
Please use camelCase

Reply:
:thinking: I received several comments about camelCase instead of snake_case. 
Can't we had this requirement to linter ? ^^' This way we won't lose time with review ^^'